### ISSUE TITLE
it seems can not works on micro bit v2

### ISSUE THREAD

**Issue by goforwork** [PhonicCanine/vscode-microbit#7]

**Comment by MrMazzone** [comment]
I can also confirm, Micro:bit v2 does not seem to work.

**Comment by PhonicCanine** [comment]
How do you currently build to a v2? (I only currently have access to v1)
If there's a command line strategy (from my brief search, it seems as if the v2 is not currently supported by uflash, at the very least), then I'm very happy go and implement it.

**Comment by MrMazzone** [comment]
Looks like they very recently starting working on it https://github.com/ntoll/uflash/issues/67

I will look a bit closer at this also.

**Comment by MrMazzone** [comment]
I have changed my local copy of uflash.py to the one discussed on the uflash issue and Micro:bit v2 worked great! 
https://github.com/mu-editor/mu/blob/master/mu/contrib/uflash.py

The extension should just work once uflash is officially updated.

**Comment by osterhoutan** [comment]
Yeah @MrMazzone 's solution did not work out for me, maybe I grabbed the wrong version of the `uflash.py` from the mu-editor repo or something in my conda env is not working properly but this definitely still does not allow me to build and flash anything onto my v2.

**Comment by MrMazzone** [comment]
Hi @osterhoutan. This is the uflash.py I have.  Still working great for my v2. 
https://1drv.ms/u/s!Ap8cS8TATIfLvNZJboX9arssZe_Okw?e=UH1Hyd
